zerver/views/message_send.py

```diff
@@ -196,10 +196,10 @@ def send_message_backend(request: HttpRequest, user_profile: UserProfile,
             # Also, mypy can't detect that a single-item
             # list populated from a Union[int, str] is actually
             # a Union[Sequence[int], Sequence[str]].
```

Comment:
Do we get to adjust this comment too?

Reply:
The comment still applies, now as an explanation for why we need an `if` with seemingly identical branches.